### DISCUSSION TITLE
[Question] Laravel 8 : Can this package be used with Laravel 8?

### DISCUSSION THREAD

**Discussion by domthomas-dev** [jenssegers/laravel-mongodb#2081] [General] [answered]
This package can be used with Laravel 6 in standard
This packages can be used with Laravel 7 with an alpha version (Thanks for that !)

Can this package be used with Laravel 8 ? and How ?

Thanks.

**Comment by divine** [comment, marked as answer]
Hello,

Let's wait release date, after that we could _probably_ comment about when it will be supported.

Sorry for late reply.

-----
**Update:**

Laravel 8 support is on the way, please  follow up this [PR](https://github.com/jenssegers/laravel-mongodb/pulls)

**Reply by divine** [reply]
We've released new version! Enjoy!

**Comment by domthomas-dev** [comment]
And you have an idea for the release date ?

**Comment by divine** [comment]
> And you have an idea for the release date ?

I don't know, maybe one or two weeks after laravel 8 release but no guarantees.

**Comment by scorgn** [comment]
Wondering if there's some update for this

**Reply by divine** [reply]
Hello,

There will be update in a few days.

Thanks!

**Reply by scorgn** [reply]
Perfect thank you! Side note, do you know if there are any code changes required for the support or if it is just the dependencies that need to be updated? If it's just the dependencies I could fork the repository, update the dependencies, and then require the fork just so I am ready for the update when it happens.

I tried cloning the repo and running tests to see if they would still pass with just the dependencies updated but I couldn't get them to pass even with the original develop branch. I'm not sure if I'm doing something wrong or if I need to update environment variables or anything. I just ran `docker-compose up` and when it was finished ran `./vendor/bin/phpunit`.

Thanks in advance!

**Reply by divine** [reply]
There will no breaking neither code changes will be required in your project.

However for this library itself it's not just dependencies upgrade, code changes will be required as well, it will not affect your project in any away.

**Reply by scorgn** [reply]
Thanks for the prompt response! I will await the changes then.

I'm happy to help contribute for the changes if you guys need an extra hand, if I can get the tests passing at least.

**Reply by divine** [reply]
> Thanks for the prompt response! I will await the changes then.
> 
> I'm happy to help contribute for the changes if you guys need an extra hand, if I can get the tests passing at least.

Looks like mysql port wasn't exposed within docker. Could you please try [this](https://github.com/jenssegers/laravel-mongodb/pull/2096/commits/ca9fa629ee74777e76714fc5899521f19e50b09b) change and running tests again?

Thanks!

**Reply by scorgn** [reply]
I will try that after work today!

**Comment by hekmatinasser** [comment]
hi
laravel 8 release at 8 sep. Can this package be used with Laravel 8 ?

**Reply by scorgn** [reply]
The comments on here have already answered this. Right now it does not but they will have an update in a few days

**Comment by rezkym-archive** [comment]
Is there any recent good news for laravel 8?

**Reply by divine** [reply]
Follow this [PR](https://github.com/jenssegers/laravel-mongodb/pull/2108)
